### ISSUE TITLE
Accessability Test || Webapp || On attendance page -the Present,absent and late buttons are narrated as buttons - not specified their functionality.

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#368]
**Describe the bug**
 On the attendance page - Present, absent and late buttons are narrated as buttons - not specified their functionality.

**To Reproduce**
1. Go to Attendance page
2. Click on Record Attendance button
3. Click on Add a new occurrence button
4. Select date and time
5. On the attendance recording page - present icon, absent icon and late icons are narrated as buttons only

**Expected behavior**
On the attendance page - Present, absent and late buttons are expected to narrated as button present, button absent and button late respectively

**Screenshots**

https://user-images.githubusercontent.com/78338755/178713146-7ea81fc5-7c22-4172-8fbb-c5ec055c1dc1.mp4

**Comment by mikedawson** [comment]
@poojaustad I think this should be fixed - can you please test with the new web version from dev-mvvm-primary?

**Comment by poojaustad** [comment]
Yes it's fixed
